core/src/ast_builder/expr/is_null.rs

```diff
@@ -0,0 +1,27 @@
+use super::ExprNode;
+
+impl ExprNode {
+    pub fn is_null(self) -> Self {
+        Self::IsNull(Box::new(self))
+    }
+
+    pub fn is_not_null(self) -> Self {
```

Comment:
[comment]
Though the file name is `is_null`, it contains `is_not_null`. But I understand that making a new file for `is_not_null` seems too much.

Reply:
How about omitting the is_ suffix and putting null related stuff in the null name?

Reply:
In other cases (`ast/ Expr`)
![image](https://user-images.githubusercontent.com/2025065/177580480-a6b0b10c-52eb-4a74-9849-8edf88d7d598.png)
`negated` field is used for merging two different exprs into a single one.

However, `IsNull` and `IsNotNull` are kind of exceptions. so.. I think it would be acceptable to gather `is_null` and `is_not_null` into a single file.